src/omp.jl

```diff
@@ -0,0 +1,200 @@
+struct OMPTracer{T}
+    iteration::Vector{Int}
+    active::Vector{Vector{Int}}
```

Comment:
It occurs to me that we could use `SparseVector` instead of `active` and `activesoln`. I'm no proposing to do this, but to think about it. Do you have an opinion on this?

Reply:
Both `omp` and `bpdual` now use `SparseVector`. @cortner Could you please confirm that everything is in order so I can proceed with merging this PR?